README.md

```diff
@@ -101,19 +101,34 @@ You can debug your Serverless Application's Lambda Function locally using the Co
 
 These steps are relevant for javascript lambda functions, and assume you have a properly configured Serverless Application template file.
 
-* Define the payload that will be passed into your lambda function. Create a file next to your `template.yaml` called `event.json` and put an empty JSON object in the file:
+* Create a file that defines the payload that will be passed into your lambda function. Create a file next to your `template.yaml` called `event.json` and put an empty JSON object in the file:
 
     ```javascript
     {
     }
     ```
 
+    After you have debugging set up, you can return to this file and change its contents to be the payload you want to debug with.
+
 * Define a task responsible for running the lambda function locally using SAM CLI. From the **Command Palette**, select `Tasks: Configure Task`.
 
   * If your workspace does not have a `tasks.json` file, you will be prompted further. Select `Create tasks.json file from template`, then `Others`. Delete the echo task from the newly created file.
   * If your workspace does have a `tasks.json` file, select one of your existing task definitions, or select `Open tasks.json file` to open the file.
 
-* Create an entry in the `tasks` array
+* You should be looking at a `tasks.json` file that looks similar to this:
+
+    ```json
+    {
+        "version": "2.0.0",
+        "tasks": [
+
+            <In here you may have 0 or more task entries>
```

Comment:
This feels verbose/oddly specific, and not particularly relevant. As a reader, I'm thinking "What am I supposed to do with this information?" Consider replacing with one of the following:

1. `// ...`
2. `// ...existing tasks...`
3. `// You might have existing tasks here.`

or similar. My preference would be for `// ...` or removing the message entirely.

Reply:
I deliberately avoided comments because that looks more like something you could expect to legitimately be present. This is meant to give you a good idea of what you should see in the file, and if you're following along with the instructions, you'll have a good understanding of what this represents when you're looking at your own file.

Reply:
@awschristou That's fine, I'm not worried about `// ...` vs `<...>`. I'm more concerned with simplifying or removing the message.